pools.json

```diff
@@ -50,39 +29,17 @@
             "t1KyRNrgUUt8wPZso36LucEs8Q2hYNe2kAA"
         ]
     },
-    {
-        "poolName": "EquiGems!",
-        "url": "https://equigems.online",
-        "searchStrings": [
-            "t1asFDxJpXgc5UyAsXVxEzsiJAdREP6YLh4"
-        ]
-    },
     {
         "poolName": "Equihash.pro",
         "url": "https://equihash.pro/",
         "searchStrings": [
             "t1VoiZk4sdvzstMbDTVgYDzBgBEL3dCPuUZ",
             "t1fp4WH4QL3zJa3zeFFu2PcT34tbMma5HUc",
-            "t1bkSfToeGPMjnm6AvEhjGZdFvRZvksD4SX",
-            "t1LFTQYatpcPLqmkoW9kDL1ZmQK2jSqMSfk"
```

Comment:
The last one deleted is currently correct pool address
https://github.com/zelcash/insight-api/commit/05da98b1725a3347a1a215589aca0ed84c13a560#diff-e20a9f7abfd0a7151b56854fd4cf2f8fR67